### ISSUE TITLE
Request other source code in the DropMax paper

### ISSUE THREAD

**Issue by XueZ-phd** [haebeom-lee/dropmax#6]
Hi, Hae Beom Lee:

I am reading your paper NeurIPS18: "DropMax", which is indeed a great research work!

Can you open source the code for the other models mentioned in the paper? Specifically, resnet34 on the CIFAR-10/CIFAR-100 and resnet18 on the AWA/CUB-200-2011, although I can download and retrain these backbone networks, I don't know how you use data preprocessing, batch size, learning rate and learning strategies etc., so for a fair comparison I would like to request those codes.

Also, if you can't find those codes, can I report the corresponding results from your paper and cite them in my future paper?

Looking forward to your response!

I wish you all the best.

Xue

**Comment by haebeom-lee** [comment]
Hi Xue!

I really thank you for having interest in our paper. And I'm extremely sorry for late reply, I just forgot...
Yes, it is completely okay to report the numbers in my paper. It will make things easier! (the codes are so messy..) And also thanks a lot for considering to cite our paper for your future paper.

Best regards,
Hae Beom Lee

**Comment by XueZ-phd** [comment]
Hi, Hae Beom Lee,

Thanks for your reply, it is a little late but very  helpful.

 

Before your response, I rerun your code released on github including base softmax and dropmax, in the future, I am intend to report the data in your paper as well as those I get by rerunning your code.

 

Finally, Thanks for your codes and your reply!

 

Best wishes, 

 

Xue, zhang

 

发件人: noreply@github.com <noreply@github.com> 代表 Hae Beom Lee
发送时间: 2020年12月16日 0:01
收件人: haebeom-lee/dropmax <dropmax@noreply.github.com>
抄送: XueZ-phd <zxue2019@zju.edu.cn>; Author <author@noreply.github.com>
主题: Re: [haebeom-lee/dropmax] Request other source code in the DropMax paper (#6)

 

Hi Xue!

I really thank you for having interest in our paper. And I'm extremely sorry for late reply, I just forgot...
Yes, it is completely okay to report the numbers in my paper. It will make things easier! (the codes are so messy..) And also thanks a lot for considering to cite our paper for your future paper.

Best regards,
Hae Beom Lee

**Comment by XueZ-phd** [comment]
Hi Hae Beom Lee!




I am trying to re-implement your codes on CIFAR and CUB, but get worse results, I encounter some problems, and I am eager for your help:

1. Beacause I don't have the pretrained checkpoint files for resnet34 as well as resnet18 (they are the base networks in your DropMax papaer), but resnet32 and resnet50 are available, following experimental settings of Kaiming He on CIFAR, for a fair comparison with DropMax, I have to re-implement your results on CIFAR/resnet32 and CUB/resnet50.

2. Using baseline softmax strategy, I get accuracy (5 runs) of 68.920% ± 0.25 on CIFAR100/resnet32 and 66.213% ± 0.224 on CUB200_2011/resnet50, however， frustratingly, when equipped with DropMax, I get worse accuracy of 67.99% ± 0.30 on CIFAR100/resnet32 and 'nan' on CUB200_2011/resnet50, seperately.




Therefore, I am writing to request your original codes on CIFAR/resnet34 as well as on CUB/resnet18. As you said before, your codes are messy, to save your time, I upload baseline softmax version on CIFAR100/resnet32 as well as on CUB200_2011/resnetV2_50, baseing on them, I hope that you can implement your DropMax and generously share your implementations with me.




I am looking forward to your reply, thanks!

Best wishes,

Xue Zhang



-----原始邮件-----
发件人:"Hae Beom Lee" <notifications@github.com>
发送时间:2020-12-16 00:00:30 (星期三)
收件人: haebeom-lee/dropmax <dropmax@noreply.github.com>
抄送: XueZ-phd <zxue2019@zju.edu.cn>, Author <author@noreply.github.com>
主题: Re: [haebeom-lee/dropmax] Request other source code in the DropMax paper (#6)






Hi Xue!

I really thank you for having interest in our paper. And I'm extremely sorry for late reply, I just forgot...
Yes, it is completely okay to report the numbers in my paper. It will make things easier! (the codes are so messy..) And also thanks a lot for considering to cite our paper for your future paper.

Best regards,
Hae Beom Lee
